mattermost-plugin/server/boards/boardsapp_util.go

```diff
@@ -80,8 +80,12 @@ func createBoardsConfig(mmconfig mm_model.Config, baseURL string, serverID strin
 		showFullName = *mmconfig.PrivacySettings.ShowFullName
 	}
 
+	serverRoot := baseURL + "plugins/focalboard"
```

Comment:
```suggestion
	serverRoot := baseURL + "/plugins/focalboard"
```

Reply:
I knew I wanted you to look for a reason...nice catch.